src/main/java/com/report/portal/business/steps/BaseSteps.java

```diff
@@ -9,9 +9,14 @@
 public class BaseSteps {
 
     private final BasePage basePage = new BasePage();
+    private LoginPage loginPage;
+
+    public BaseSteps openInitialPage() {
+        loginPage = Selenide.open("http://localhost:8080/ui/#login", LoginPage.class);
```

Comment:
Maybe it's better to storethe link in an aditional file or move that link to a variable?